### PR TITLE
Link to docs & Ansible 2.5 is required

### PR DESCRIPTION
**Pull request by gundalow**
* Link to the various user & development guides
* Specifies that Ansible 2.5.0 is the required version
* Correct link to action_plugin docs as they have moved

### CODE REVIEW

**Review by trishnaguha — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,9 +9,17 @@ Any open bugs and/or feature requests are tracked in [GitHub issues](../../issue
 
 Interested in contributing to this role, please see [CONTRIBUTING](CONTRIBUTING.md)
 
+## Documentation
+
+* User guide: [[Parser Directives]](docs/directives/parser_directives.md)
+* User guide: [[Template Directives]](docs/directives/template_directives.md)
```
Comment (privateip) [inline README.md]: Should not be part of the this release.. this is coming in a subsequent release

File: README.md [hunk 2]
```diff
@@ -30,7 +38,7 @@ None
 
 The following is a list of modules that are provided by this role.
 
-* `cli` [[source]](library/cli.py)
+* `cli` [[source]](action_plugin/cli.py)
 * `get_config` [[source]](library/get_config.py)
```
Comment (privateip) [inline README.md]: ~Not part of this release~

Ignore this

**Review by privateip — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (gundalow) [reply README.md]: Removed

**Comment by gundalow** [comment]
@privateip said he's happy with this, so merging

**Pull request merged**